packages/rancher-istio/charts/values.yaml

```diff
@@ -71,103 +44,19 @@ sidecarInjectorWebhook:
   objectSelector:
     enabled: true
     autoInject: true
+  rewriteAppHTTPProbe: true
 
 global:
-  overlayFile: ""
-  accessLogFile: ""
-  disablePolicyChecks: true
-  defaultConfig:
-    proxyMetadata: {}
-  enableAutoMtls: true
-  arch:
-    amd64: 2
-    ppc64le: 2
-    s390x: 2
-  configValidation: true
-  controlPlaneSecurityEnabled: false
-  defaultNodeSelector: {}
-  defaultPodDisruptionBudget:
-    enabled: true
-  defaultResources:
-    requests:
-      cpu: 10m
-  enableHelmTest: false
-  imagePullPolicy: IfNotPresent
-  imagePullSecrets: []
   istiod:
     enableAnalysis: false
     enabled: true
-  logging:
-    level: "default:info"
   meshExpansion:
     enabled: false
     useILB: false
-  meshNetworks: {}
-  mountMtlsCerts: true
   multiCluster:
     enabled: false
     clusterName: ""
```

Comment:
for now lets just remove non-image overrides to simplify this, we can add back later